app.py

```
@@ -1,4 +1,209 @@
+    global facial_recognition_process
+    if facial_recognition_process and facial_recognition_process.poll() is None:
+        # Terminate the facial recognition process
+        facial_recognition_process.terminate()
+        facial_recognition_process = None
+        return "Facial recognition stopped successfully.", 200
+    else:
+        return "No facial recognition process is currently running.", 400
+
+
+if __name__ == '__main__':
+    app.run(debug=True, host='0.0.0.0', port=5000)
```

Ruff (0.8.2)

Use of `debug=True` in Flask app detected

(S201)